src/transport_ofi.c

```diff
@@ -999,7 +998,7 @@ static int shmem_transport_ofi_ctx_init(shmem_transport_ctx_t *ctx, int id)
 
     struct fi_cq_attr cq_attr = {0};
     cq_attr.format = FI_CQ_FORMAT_CONTEXT;
-    cq_attr.size = shmem_transport_ofi_queue_slots;
+    cq_attr.size = 0; /* Size chosen by provider */
```

Comment:
@davidozog We can actually delete this line, since it's already set to 0 on line 999.  Should it go?

Reply:
Definitely, that would be a good choose.  :wink:

Reply:
Done.  :)